script.js

```diff
@@ -0,0 +1,398 @@
+mobileFourthImgSec.appendChild(mobileFourthImage);
+mobileFourthGrid.appendChild(mobileFourthImgSec);
+mobileFourthGrid.appendChild(mobileFourthDiv);
+mobileFeature.appendChild(mobileFourthGrid);
+
+const mobileFifthGrid = document.createElement('div');
+const mobileFifthImgSec = document.createElement('div');
+const mobileFifthImage = document.createElement('img');
+const mobileFifthDiv = document.createElement('div');
+const mobileFifthName = document.createElement('h2');
+const mobileFifthPosition = document.createElement('div');
+const mobileFifthText = document.createElement('p');
+mobileFifthText.innerText = mobileSpeakers.FifthSpeaker.text;
+mobileFifthPosition.innerText = mobileSpeakers.FifthSpeaker.position;
+mobileFifthPosition.className = 'orange';
+mobileFifthName.innerText = mobileSpeakers.FifthSpeaker.name;
+mobileFifthDiv.appendChild(mobileFifthName);
+mobileFifthDiv.appendChild(mobileFifthPosition);
+mobileFifthDiv.appendChild(mobileFifthText);
+mobileFifthDiv.className = 'div-g';
+mobileFifthGrid.className = 'j-grid';
+mobileFifthImage.src = mobileSpeakers.FifthSpeaker.image;
+mobileFifthImgSec.appendChild(mobileFifthImage);
+mobileFifthGrid.appendChild(mobileFifthImgSec);
+mobileFifthGrid.appendChild(mobileFifthDiv);
+mobileFeature.appendChild(mobileFifthGrid);
+
+const mobileSixthGrid = document.createElement('div');
+const mobileSixthImgSec = document.createElement('div');
+const mobileSixthImage = document.createElement('img');
+const mobileSixthDiv = document.createElement('div');
+const mobileSixthName = document.createElement('h2');
+const mobileSixthPosition = document.createElement('div');
+const mobileSixthText = document.createElement('p');
+mobileSixthText.innerText = mobileSpeakers.SixthSpeaker.text;
+mobileSixthPosition.innerText = mobileSpeakers.SixthSpeaker.position;
+mobileSixthPosition.className = 'orange';
+mobileSixthName.innerText = mobileSpeakers.SixthSpeaker.name;
+mobileSixthDiv.appendChild(mobileSixthName);
+mobileSixthDiv.appendChild(mobileSixthPosition);
+mobileSixthDiv.appendChild(mobileSixthText);
+mobileSixthDiv.className = 'div-g';
+mobileSixthGrid.className = 'j-grid';
+mobileSixthImage.src = mobileSpeakers.SixthSpeaker.image;
+mobileSixthImgSec.appendChild(mobileSixthImage);
+mobileSixthGrid.appendChild(mobileSixthImgSec);
+mobileSixthGrid.appendChild(mobileSixthDiv);
+mobileFeature.appendChild(mobileSixthGrid);
+
+const menuBtn = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.mobile-nav');
+
+menuBtn.addEventListener('click', () => {
+  menuBtn.classList.toggle('is-active');
+  mobileMenu.classList.toggle('is-active');
+});
+
+const take = document.querySelector('.take');
+const don = document.querySelector('.don');
+const MobileAbout = document.getElementById('mobile-about');
+const mobileHome = document.getElementById('mobile-home');
+take.addEventListener('click', () => {
+  MobileAbout.style.display = 'block';
+  mobileHome.style.display = 'none';
+});
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that when the `about` navigation link in the mobile version is clicked, the correct page is rendered and the sidebar is removed ☺️.